app/models.py

```diff
@@ -130,6 +130,7 @@ class Person(SearchableMixin, db.Model):
     college_code = db.Column(db.String)
     leave = db.Column(db.Boolean)
     eli_whitney = db.Column(db.Boolean)
+    visiting_international = db.Column(db.Boolean)
```

Comment:
Can we just call this `visitor` for brevity?

Reply:
Because every single JSON payload will now include this property, it's worth keeping it short. If you can think of another better name, I'm open to that.